### PR TITLE
Ic/chapter06

### PR DESCRIPTION
**Pull request by ismaelcv**

### CODE REVIEW

File: chapter06/docker-compose.yml [hunk 1]
```diff
@@ -123,7 +112,7 @@ services:
     <<: *airflow-common
     command: webserver
     ports:
-      - "8080:8080"
+      - "8082:8080"
```
Comment (krisgeus) [inline chapter06/docker-compose.yml:115]: Why do we need a different port here. All other compose files use 8080

File: chapter06/docker-compose.yml [hunk 2]
```diff
@@ -64,25 +63,13 @@ x-airflow-common:
     AIRFLOW__API__AUTH_BACKENDS: 'airflow.api.auth.backend.basic_auth,airflow.api.auth.backend.session'
     AIRFLOW__SCHEDULER__DAG_DIR_LIST_INTERVAL : 60
     AIRFLOW__CORE__TEST_CONNECTION: 'Enabled'
-
```
Comment (krisgeus) [inline chapter06/docker-compose.yml:67]: Why is this removed. I would like to keep the docker compose files for all chapters as much aligned as possible. Also as much as possible equal to the official docker compose file. I think I only changed minor things like the data volume and the disabling of the examples.

File: chapter06/docker-compose.yml [hunk 3]
```diff
@@ -254,21 +229,19 @@ services:
         fi
         mkdir -p /sources/logs /sources/dags /sources/plugins
         chown -R "${AIRFLOW_UID}:0" /sources/{logs,dags,plugins}
-        chown -R "${AIRFLOW_UID}:0" /data
```
Comment (krisgeus) [inline chapter06/docker-compose.yml:257]: I;d like to keep this so the chapters compose files are equal as much as possible

File: chapter06/dags/12_Inter_DAG_dependencies.py [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+import pendulum
+from airflow import DAG
+from airflow.operators.empty import EmptyOperator
+from airflow.operators.python import PythonOperator
+from airflow.operators.trigger_dagrun import TriggerDagRunOperator
+
+# ================================================ EXAMPLE 1 =================================================
+
+with DAG(
+    dag_id="figure_6_17_example_1_dag_1",
```
Comment (krisgeus) [inline chapter06/dags/12_Inter_DAG_dependencies.py:15]: Rename the dags to adhere to naming of the file convention?

File: chapter06/dags/15_ExternalTaskSensor_non_aligned_schedules.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
+"""
+    Listing: 6.6
+"""
+
+import pendulum
+from airflow import DAG
+from airflow.operators.empty import EmptyOperator
+from airflow.sensors.external_task import ExternalTaskSensor
+
+with DAG(dag_id="dag1", schedule="0 16 * * *", start_date=pendulum.today("UTC").add(days=-3)):
```
Comment (krisgeus) [inline chapter06/dags/15_ExternalTaskSensor_non_aligned_schedules.py:10]: Use section number (15) in dag name so it aligns in the UI close to where it is used.

File: chapter06/docker-compose.yml (hunk 1 above)
Comment (ismaelcv) [reply chapter06/docker-compose.yml:115]: indeed

File: chapter06/docker-compose.yml (hunk 2 above)
Comment (ismaelcv) [reply chapter06/docker-compose.yml:67]: i agree with you. I will bring it back closest to the latest docker compose version

File: chapter06/dags/10_multiple_tasks_metrics.py [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+from pathlib import Path
+
+import pendulum
+from airflow import DAG
+from airflow.operators.empty import EmptyOperator
+from airflow.sensors.python import PythonSensor
+
+
+def _wait_for_supermarket(supermarket_id_):
```
[the remark this answers is not in the document]
Comment (ismaelcv) [reply chapter06/dags/10_multiple_tasks_metrics.py:13]: agree

File: chapter06/docker-compose.yml (hunk 3 above)
Comment (ismaelcv) [reply chapter06/docker-compose.yml:257]: indeed

File: chapter06/dags/15_ExternalTaskSensor_non_aligned_schedules.py (hunk 5 above)
Comment (ismaelcv) [reply chapter06/dags/15_ExternalTaskSensor_non_aligned_schedules.py:10]: agree

File: chapter06/dags/12_Inter_DAG_dependencies.py (hunk 4 above)
Comment (ismaelcv) [reply chapter06/dags/12_Inter_DAG_dependencies.py:15]: agree

File: official-airflow-docker-compose.yml [hunk 7]
```diff
@@ -119,7 +119,7 @@ services:
     <<: *airflow-common
     command: webserver
     ports:
-      - "8080:8080"
+      - "8082:8080"
```
Comment (krisgeus) [inline official-airflow-docker-compose.yml:122]: Could you revert this change so airflow is running on 8080 for all chapters

**Review by krisgeus — APPROVED** [review]
